modules/sample/src/main/resources/server3.yaml

```diff
@@ -0,0 +1,73 @@
+swagger: "2.0"
+info:
+  title: Whatever
+  version: 1.0.0
+host: localhost:1234
+schemes:
+  - http
+definitions:
+  SecureAddress:
+    type: object
+    properties:
+      line1:
+        type: string
+      line2:
+        type: string
+      line3:
+        type: string
+#  UnauthorizedError:
+#    headers:
+#      WWW_Authenticate:
+#          type: string
+
+paths:
+  /secure/address:
+    get:
+      operationId: getSecureAddresses
+      x-jvm-package: secure.addresses
+      security:
+        - basicAuth: []
+      produces:
+        - application/json
+      responses:
+        '200':
+          type: array
+          items:
+            $ref: '#/definitions/SecureAddress'
+#        '401':
```

Comment:
Was this commented out due to #179? If so, the fix has been merged, hopefully this can get uncommented